src/include/txn.i

```diff
@@ -10,58 +10,111 @@ static inline int __wt_txn_id_check(WT_SESSION_IMPL *session);
 static inline void __wt_txn_read_last(WT_SESSION_IMPL *session);
 
 #ifdef HAVE_TIMESTAMPS
+#if WT_TIMESTAMP_SIZE == 8
```

Comment:
I added an additional macro `WT_TIMESTAMP_UINT64` because it made it easier to do testing, that is,  I could keep `WT_TIMESTAMP_SIZE == 8` (which our test programs expect), but switch to using the `uint8_t` array instead of the `uint64_t` value. It was just a testing hack, usefulness depends on your plans for testing.